docs/internal.md

```diff
@@ -31,9 +40,21 @@ Host agent:
 - `ha.stdout.log`: hostagent stdout (JSON lines, see `pkg/hostagent/api.Events`)
 - `ha.stderr.log`: hostagent stderr (human-readable messages)
 
-## Cache directory (`~/Library/Caches/lima/download/by-url-sha256/<SHA256_OF_URL>`)
+## Lima cache directory (`~/Library/Caches/lima`)
+
+Currently hard-coded to `~/Library/Caches/lima` on macOS.
```

Comment:
@jandubois Do you want this directory to be separated out for Rancher Desktop too?

If so, what should be the env var for this?  Can we (ab)use `$XDG_CACHE_HOME`? Or should we define a new env var like `$LIMA_CACHE`?

Reply:
I don't know yet, I suspect that Rancher Desktop may not be using the lima cache (e.g. it will include the ISO inside the app). But if we do need to use it, it should be separate, so a user running `limactl prune` will not clean out the RD cache.

Personally I would prefer `$LIMA_CACHE` over using `$XDG_CACHE_HOME`, which feels alien on macOS...

Reply:
Or maybe we can just use `~/Library/Caches/lima/<hash of LIMA_HOME>` so that we do not need to care about multiple env vars.

Reply:
I would prefer to use a separate directory. E.g. if a user runs into a problem with lima and manually removes both `~/.lima` and `~/Library/Caches/lima` to start over with a fresh install, it would not be obvious that this would also remove the RD cache as well.

Reply:
Opened https://github.com/AkihiroSuda/lima/issues/63 for further discussion